manualdirectdebit.php

```diff
@@ -166,3 +166,103 @@ function manualdirectdebit_civicrm_navigationMenu(&$menu) {
     _manualdirectdebit_civix_insert_navigation_menu($menu, 'Administer/' . $directDebitMenuItem['name'], $menuItem);
   }
 }
+
+/**
+ * Implements hook_civicrm_postProcess().
+ *
+ */
+function manualdirectdebit_civicrm_postProcess($formName, &$form) {
```

Comment:
it is not a good idea to write bussnies logic inside hook impelemenaton, better to move it to seperate class, there are many ways to achieve that, one example is what we followed here : https://github.com/compucorp/uk.co.compucorp.membershipextras/blob/develop/membershipextras.php#L132